libs/remix-core-plugin/src/lib/compiler-content-imports.ts

```diff
@@ -116,8 +116,7 @@ export class CompilerImports extends Plugin {
     * @param {String} targetPath - (optional) internal path where the content should be saved to
     * @returns {Promise} - string content
     */
-  async resolveAndSave (url, targetPath) {
-    if (url.indexOf('remix_tests.sol') !== -1) return remixTests.assertLibCode
+  async resolveAndSave (url, targetPath, testRunner = null) {
```

Comment:
this can be rollbacked (and above also)

Reply:
ah yes. but `if (url.indexOf('remix_tests.sol') !== -1) return remixTests.assertLibCode` is not needed anymore i think as we create files in FE